### ISSUE TITLE
Debug mode breaks Icecast login

### ISSUE THREAD

**Issue by turlando** [turlando/airhead#11]
* How to reproduce
   1. Set `Debug = True` in `airhead.ini`.
   2. Run app.py

* Backtrace
```
 * Running on http://0.0.0.0:8080/ (Press CTRL+C to quit)
 * Restarting with stat
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib64/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/home/turlando/Projects/airhead/airhead/transmitter.py", line 37, in run
    with shouty.connect(**self.params) as connection:
  File "/usr/lib64/python3.6/contextlib.py", line 82, in __enter__
    return next(self.gen)
  File "/home/turlando/Projects/airhead/env/lib/python3.6/site-packages/shouty/utils.py", line 8, in connect
    cn.open()
  File "/home/turlando/Projects/airhead/env/lib/python3.6/site-packages/shouty/connection.py", line 30, in decorated
    err_name, err_str))
Exception: Failed open
Error code: NOLOGIN
Error description: Login failed

 * Debugger is active!
 * Debugger PIN: 253-663-331
```

**Comment by turlando** [comment]
#13 will resolve this issue (hopefully?). Changing priority to low.

**Comment by turlando** [comment]
Fixed in [ef984cd](https://github.com/turlando/airhead/commit/ef984cd37c8fdf805be85be4c5baf7e9fafcef2c).
